src/main/java/org/openshift/jenkins/plugins/openshiftlogin/OpenShiftOAuth2SecurityRealm.java

```diff
@@ -257,11 +258,13 @@ public void initialize(HttpRequest request) throws IOException {
                                 }
                             });
                     GenericUrl url = new GenericUrl(serverPrefix + "/oapi/v1/users/~");
-
+                    
+                    // do self-sar here to see if you have the "admin" or "view" verb in the "build.openshift.io" api group on the "jenkins" resource 
```

Comment:
OK that helped - thanks.  Inching closer ... seems like I'm close.  Adding both scopes (had to otherwise the get of the `User` failed without `user:info`) I new get a response from the POST of subjectaccessreview instead of the Java access exception, but it still failed.

The http response was:
```
GGM response from post {"kind":"SubjectAccessReviewResponse","apiVersion":"v1","namespace":"test","allowed":false,"reason":"scopes [user:check-access user:info] prevent this action; User \"test\" cannot \"admin\" \"build.openshift.io.jenkins\" with name \"\" in project \"test\""}
```

That looks like a ref to the `build.openshift.io` api group and `jenkins` resource you mentioned before.

I added the `admin` role to the test user and logged out / logged back in, but same error.

Wasn't optimistic but I tried with no luck `oadm groups add-user ..` and either `build.openshift.io` or `build.openshift.io.jenkins` ... so as suspected api group does not correspond to `oadm groups`.

What is the next bit(s) to turn on here?  Should I go `user:full` ?